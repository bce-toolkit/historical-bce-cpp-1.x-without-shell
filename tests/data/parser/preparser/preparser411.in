XeF4+H2O=XeO3+Xe+O2+HF