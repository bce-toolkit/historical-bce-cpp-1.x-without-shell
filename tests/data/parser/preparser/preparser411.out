XeF4 0 0
H2O 0 0
XeO3 1 0
Xe 1 0
O2 1 0
HF 1 0
